src/frontend/komponenter/Felleskomponenter/Venstremeny/sider.ts

```diff
@@ -110,13 +110,18 @@ export const visSide = (side: ISide, åpenBehandling: IBehandling, harOpplysning
 export const finnSideForBehandlingssteg = (
     steg: BehandlingSteg,
     opplysningsplikt: IOpplysningsplikt | undefined
-) => {
+): ISide | undefined => {
     if (hentStegNummer(steg) >= hentStegNummer(BehandlingSteg.SEND_TIL_BESLUTTER)) {
         return sider.VEDTAK;
     } else if (opplysningsplikt && opplysningsplikt.status === OpplysningspliktStatus.IKKE_SATT) {
         return sider.OPPLYSNINGSPLIKT;
     }
-    return Object.values(sider).find((side: ISide) => side.steg === steg);
+
+    const sideForSteg = Object.entries(sider)
+        .filter(([sideId, _]) => sideId !== SideId.OPPLYSNINGSPLIKT)
+        .find(([_, side]) => side.steg === steg);
+
+    return sideForSteg ? sideForSteg[1] : undefined;
```

Comment:
[1] for å hente value?

Reply:
mhm, litt som kotlin pairs. Vi så litt etter mer elegante måter, men fant ikke noe umiddelbart. 
Hvis det ikke finnes mer intuitive måter bør vi kanskje vurdere om record er det vi trenger?

Reply:
Nja, problemet her er vel at vi kjører entries på recorden. Man kunne f.eks. gjort keys også hentet ut verdien på den måten, men smak og behag :)